scripts/add-license-headers.sh

```
@@ -135,9 +140,27 @@ add_ocaml_header() {
+            # and it never said WHICH file. Refuse explicitly and name it,
+            # because a duplicate line is itself the defect to fix (four files
+            # in this repo had one), not a state to paper over by taking the
+            # first match.
+            local matches
+            matches=$(head -"$header_end_line" "$file" | grep "SPDX-FileCopyrightText:" | grep -ci "$contributor_email")
+            if [ "$matches" -gt 1 ]; then
+                echo -e "${RED}DUPLICATE${NC}: $file has $matches SPDX-FileCopyrightText lines for <$contributor_email>; remove the extra one (this script cannot choose between them)" >&2
+                DUPLICATE_COUNT=$((DUPLICATE_COUNT + 1))
+                return
+            fi
             local contributor_line=$(head -"$header_end_line" "$file" | grep "SPDX-FileCopyrightText:" | grep -i "$contributor_email")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)